codec-http2/src/main/java/io/netty/handler/codec/http2/hpack/Decoder.java

```diff
@@ -0,0 +1,568 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
```

Comment:
I think we should "just" add our license header to all the files while keep the original ones. Then we also need to modify the NOTICE file and add the license to the license directory.